docs/object-capabilities/derived-authority.md

```diff
@@ -29,17 +29,17 @@ class MyTCPConnectionNotify is TCPConnectionNotify
     _out.print("connect_failed")
 
 actor Connect
-  new create(out: OutStream, auth: AmbientAuth) =>
+  new create(out: OutStream, auth: TCPConnectAuth) =>
     TCPConnection(auth, MyTCPConnectionNotify(out), "example.com", "80")
 
 actor Main
   new create(env: Env) =>
-    Connect(env.out, env.root)
+    Connect(env.out, TCPConnectAuth(env.root))
 ```
 
-The `Main` actor authorizes the `Connect` actor by passing the ambient authority token on to it. This token is unforgeable since the `AmbientAuth` constructor is private and the only existing instance is provided by the runtime itself.
+The `Main` actor authorizes the `Connect` actor a `TCPConnectAuth` token created from ambient authority token in `env.root`. The ambient authority token is unforgeable since the `AmbientAuth` constructor is private and the only existing instance is provided by the runtime itself. The `TCPConnectAuth` token is derived from ambient authority token.
```

Comment:
```suggestion
The `Main` actor authorizes the `Connect` actor by passing a `TCPConnectAuth` token created from the ambient authority token in `env.root`. The ambient authority token is unforgeable since the `AmbientAuth` constructor is private and the only existing instance is provided by the runtime itself.
```